### PR TITLE
Use Avo

### PR DESCRIPTION
**Pull request by tpom6oh**

### CODE REVIEW

File: src/AvoDebugger.js [hunk 1]
```diff
@@ -9,12 +9,28 @@ import AvoDebuggerDrags from './AvoDebuggerDrags';
 import AvoBar from './avobar/AvoBar';
 import {eventsHaveErrors} from './utils';
 import PropTypes from 'prop-types';
+import Avo from './DebuggerAnalytics';
```
Comment (arnihermann) [inline src/AvoDebugger.js]: wondering if we should name the file Avo.js?

**Comment by arnihermann** [comment]
LGTM besides one thought

File: src/AvoDebugger.js (hunk 1 above)
Comment (tpom6oh) [reply src/AvoDebugger.js]: I was thinking to distinguish it from the Avo.js file the main project might be using.

Comment (arnihermann) [reply src/AvoDebugger.js]: Ah I see. It won't conflict so it should be ok?

**Pull request merged**